src/css/components/breadcrumbs.scss

```diff
@@ -3,25 +3,31 @@
 @import '../core.scss';
 
 .breadcrumbs {
-  font-size: $sans-s3;
-  line-height: $lh-single;
-  margin: 0 $grid-3 $grid-3;
+  @extend %fw400;
```

Comment:
Because we didn't come to a conclusion about whether we wanted to use the current utility classes, I don't really want to see code extended from them yet. This means here and elsewhere in this PR.